pkg/minikube/cluster/cluster.go

```diff
@@ -42,12 +43,12 @@ func init() {
 }
 
 // Bootstrapper returns a new bootstrapper for the cluster
-func Bootstrapper(api libmachine.API, bootstrapperName string) (bootstrapper.Bootstrapper, error) {
+func Bootstrapper(api libmachine.API, bootstrapperName string, cc config.ClusterConfig, n config.Node) (bootstrapper.Bootstrapper, error) {
```

Comment:
Please add a TODO(#xxx) to remove node as an argument, and/or clarify in the docstring what the node is doing here.